ember-cli-build.js

```diff
@@ -1,22 +1,22 @@
-/*jshint node:true*/
+/* eslint-env node */
 /* global require, module */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
+let EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
-  var app = new EmberAddon(defaults, {
+  let app = new EmberAddon(defaults, {
     jscsOptions: {
       enabled: true,
-      testGenerator: function(relativePath, errors) {
+      testGenerator(relativePath, errors) {
         if (errors) {
-          errors = "\\n" + this.escapeErrorString(errors);
+          errors = `\\n${this.escapeErrorString(errors)}`;
```

Comment:
oh, shit, missed that part... that was done by `eslint --fix`, but obviously this is wrong